### PR TITLE
Adds a herm gender identity

### PR DESCRIPTION
**Pull request by kcin2001**
shi/hir pronouns basically

### CODE REVIEW

**Comment by kcin2001** [comment]
I dont really have the program to squash atm if it cant be dont with just git bash but as far as i can tell it works and i just wanted some people's commonly used pronouns to work since most herms tend to use shi/hir and i felt that it would be nice for the game to match that

**Comment by kcin2001** [comment]
Ill move it, i didnt really know where to put it when i did it

**Comment by kcin2001** [comment]
dammit i was working on the wrong branch

**Comment by Little-119** [comment]
I can't recall anyone ever using these outside of Chakats.

**Comment by kcin2001** [comment]
@Little-119 i have seen it a little outside of chakats i made it since i want to use it with my drake

**Comment by Screemonster** [comment]
The "custom" gender is probably unnecessary and just opens the door for shitters to start putting wacky shit in there just to make "did you assume my gender" jokes. Only way I can think of to do it is to include some custom list on every mob to definine their pronouns, which for the small percentage that'd actually use them probably isn't worth the effort.
If we're going down that route we might as well just add "they" and that covers basically all  bases. You can't go wrong referring to someone as "they". If someone accuses you of misgendering them for calling them that, then they're just being obtuse. (and my last sentence referred to this hypothetical person with them/they three times, QED)

**Comment by kcin2001** [comment]
it wouldnt be too hard to make mob specific gender definitions because overall it would be less work but is largely unnecessary

**Comment by SpadesNeil** [comment]
@Arokha looking good now?

**Comment by Little-119** [comment]
@SpadesNeil If you're talking to me with the last part, I'm just saying that in code and user interfaces and all, the 'they' set of pronouns is referred to as 'plural', even if it can be used to refer to a singular person.

**Comment by kcin2001** [comment]
@arokha just making sure that im gonna need to get the repo locally too first right?

**Comment by Arokha** [comment]
Yes. You should test this too, given that I see you did all the edits in the web editor

**Comment by kcin2001** [comment]
I do test it i just import the code via the web editor

**Comment by kcin2001** [comment]
Ill test it again later

**Comment by Little-119** [comment]
@kcin2001 If I understand what you mean by importing the code, wouldn't it be a lot easier to just edit it in Dream Maker (or better yet, something like Atom)?

**Comment by kcin2001** [comment]
I take it from dream maker to git i just dont drag and drop and i dont know what atom is

**Comment by Little-119** [comment]
Atom is GitHub's code editor program, basically. More advanced than the web editor, though. https://atom.io/
You can even, from what I have heard, compile byond eviroments in it. I personally wasn't able to make that work, though.

What do you mean you "just don't drag and drop"? If you're using git correctly you probably shouldn't be dragging and dropping anything.

**Comment by Little-119** [comment]
@Arokha Atom also has a DM highlighter, with one of the DM packages being a port of Sublime's DM highlighter.
But regardless: No u, Aro. I'll use Atom if I want.

**Comment by SpadesNeil** [comment]
>@SpadesNeil If you're talking to me with the last part, I'm just saying that in code and user interfaces and all, the 'they' set of pronouns is referred to as 'plural', even if it can be used to refer to a singular person.

Oh I misunderstood. I usually have to smack people because "they" doesn't always mean plural even though people who don't actually know the English language claim it is and also whine about SJW ruining everything. I'd rather say "they" than memorize the never-ending encyclopedia of made-up gender pronouns because someone identifies their gender as not having a gender normally but having a gender that switches according to how the planets align. I'm not even sure if I'm making that up.

**Comment by chaoko99** [comment]
As nice as this is, I'd honestly like some level of distinction for male herms, female herms, and people who are actually willing to use shi and hir.

Because I speak English, and made-up pronouns seriously annoy me.

**Comment by kcin2001** [comment]
@chaoko if they annoy you why are you suggesting i add 2 more sets of them

**Comment by chaoko99** [comment]
I misread the intent of this pull request. I apologize.

**Comment by kcin2001** [comment]
Kk

File: code/__defines/misc_vr.dm [hunk 1]
```diff
@@ -31,4 +31,5 @@
 #define BLUE_SHIELDED 2 // Shield from bluespace teleportation (telescience)
 
 //Assistant/Visitor/Whatever
-#define USELESS_JOB "Visitor"
\ No newline at end of file
+#define USELESS_JOB "Visitor"#define HERM "herm"
```
Comment (KasparoVy) [inline code/__defines/misc_vr.dm]: :thinking: Same line?

**Comment by KasparoVy** [comment]
Wow I commented back in time, that was odd..

**Comment by kcin2001** [comment]
Yeah because i fixed it right after making it

**Pull request merged**